src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -172,34 +168,4 @@ private static DetailNode parseFile(File file) throws IOException {
             System.getProperty("file.encoding", "UTF-8"));
         return parseJavadocAsDetailNode(text.getFullText().toString());
     }
-
-    /**
-     * Creates DetailAST block comment to pass it to the Javadoc parser.
-     * @param content comment content.
-     * @return DetailAST block comment
-     */
-    private static DetailAST createFakeBlockComment(String content) {
```

Comment:
>If we are going to keep this, can another place call this method. What about TreeWalker#createBlockCommentNode?

The way things are done are similar in both methods and I can't see, not yet at least, why it can't be done.

>Since this is off topic, if we are going to keep it as a utility method with the change I asked about, it should probably be in another PR.

Yes, rather in another issue. No point holding this PR up unnecessarily as it might take time and also we will need to lower PIT coverage. So, I will create another issue, revert the movement of this method here and will grab the method by reflection for UT in this PR.